clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/InfinispanSubsystemXMLReader.java

```diff
@@ -81,10 +84,13 @@ public void readElement(XMLExtendedStreamReader reader, List<ModelNode> result)
             }
         }
 
+        // Explicitly order operations such that capabilities are defined before they are referenced
+        // This circumvents capability reference integrity issues when operations are not batched
```

Comment:
What's this about? Why aren't they "batched"? (IOW is there some problem in the kernel?)